src/cpp/proxy/engine.cpp

```diff
@@ -867,9 +867,6 @@ private slots:
 
 		log_debug("IN (retry) %s %s", qPrintable(p.requestData.method), p.requestData.uri.toEncoded().data());
 
-		if(p.retrySeq >= 0)
```

Comment:
Why are we removing stats update here, wouldn’t this mean stats would be outdated?

Reply:
these lines were moved to requestsession.cpp, where we're able to call `statsRoute()`

Reply:
specifically, inside RequestSession::startRetry, which is called below

Reply:
got it thanks for clarifying